packages/dd-trace/test/ci-visibility/exporters/agentless/writer.spec.js

```diff
@@ -23,7 +23,7 @@ describe('CI Visibility Writer', () => {
     encoder = {
       encode: sinon.stub(),
       count: sinon.stub().returns(0),
-      makePayload: sinon.stub().returns([])
+      makePayload: sinon.stub().returns(Buffer.from(''))
```

Comment:
Dear reviewer: I'm pretty sure that this `makePayload` mock should return a buffer and not an array - please verify